metals/src/main/scala/scala/meta/internal/metals/ReferenceProvider.scala

```diff
@@ -72,11 +74,36 @@ final class ReferenceProvider(
     }
   }
 
+  val workspaceMemberPriority: TrieMap[(String, String), Int] = TrieMap.empty
```

Comment:
```suggestion
  val workspaceMemberPriority: TrieMap[BuildTargetSymbol, Int] = TrieMap.empty
```
I would add a type alias or a case class here. Or you could a map per target

Reply:
Although, do we really need a build target here? Especially that what happens if you the same symbol in two different targets? Are they counted together or separate?

Maybe it doesn't make sense to differentiate, this might skew the results a bit, but maybe that's a good thing especially that the symbol is predominant in the project.

Reply:
hmm. didnt encountered cases where we need separate build targets. Removed build target from map

Reply:
Thanks!